### DISCUSSION TITLE
Incorrect local URL ( https://:443 ???)

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#469] [Q&A]
Hello, until now everything worked fine with automatic 1111+web ui in google colab but since a few hours the link generated at the end is incorrect: 

LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
Loading weights [a2a802b2] from /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/model.ckpt
Applying cross attention optimization (Doggettx).
Model loaded.
Loaded a total of 0 textual inversion embeddings.
Embeddings: 
Running on local URL:  https://:443
Connected

there is no particular error message, all the steps are working correctly, it's just the link to web ui that is 
not good

I deleted the whole sd folder and reinstalled everything, but nothing happens...is there a known bug going on, a server failure, etc etc.... ?

**Comment by TheLastBen** [comment]
use "gradio_server" for now, until the server is fixed
